assets/scss/components/_footer.scss

```diff
@@ -2,18 +2,181 @@
    #FOOTER
    ========================================================================== */
 
-footer {
-    background-color: $color-white;
+$at-sign-icon: '../../img/icons/at-sign.svg';
+
+.footer {
+    background-color: $color-secondary-50;
     border-top: 1px solid $color-white-grey;
-    padding: 3rem 0;
+    padding: 3rem;
     position: relative;
     transform: translateZ(0);
 
-    @include bp(medium) {
-        section {
+    section {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 1.5rem;
+
+        @include bp(small) {
             display: grid;
             gap: remcalc(10.5);
-            grid-template-columns: 1fr 1fr 1fr auto;
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+
+    @include bp(medium) {
+        grid-template-columns: repeat(4, 1fr);
+        padding: 5rem;
+    }
+}
+
+.footer-column {
+    width: calc(50% - 0.75rem);
+
+    &:nth-child(3) {
+        width: 100%;
+    }
+}
+
+.footer-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+
+    @include bp(medium) {
+        gap: 3rem;
+    }
+}
+
+.footer-heading {
+    color: $color-black;
+    font-family: $font-regular;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5rem;
+
+    margin-bottom: 0.75rem;
+}
+
+.footer-links {
+    margin: 0;
+}
+
+.footer-link-item {
```

Comment:
Try to use more descriptive class names, maybe `c-footer-link-wrapper` or `c-footer-item` instead of `link-item`?